### PR TITLE
Julia & Mamite, project chatbot

### PR DESCRIPTION
**Pull request by JuliaNiki**
https://chat-bot-pets.netlify.app 

We found this project pretty challenging for the real first week with Javascript, but we got an inspiration from Technigo's example of Pizza bot, knowledge from weekly materials and some help from live sessions with our school. 
We learned a lot by doing this project and we are very proud of how it looks and how it interacts with the user. 
If we had more time, we would also experiment with audio and not only videos and probably will play around with a date picker to let the user choose time and day for the procedure.

### CODE REVIEW

File: code/script.js [hunk 1]
```diff
@@ -30,18 +45,134 @@ const showMessage = (message, sender) => {
...
+  } else if (questionStep === 3) {
+    userAnswer(message)
+    setTimeout(() => priceInformation(message), 1000)
+  } else if (questionStep === 4) {
+    userAnswer(message)
+    setTimeout(() => petPayment(message), 1000)
+  } else {
+    userAnswer(message)
+    setTimeout(petBye, 1000)
+  }
```
Comment (Svempolin) [inline code/script.js:65]: Nice! I really not understanding how this works :)

File: code/script.js [hunk 2]
```diff
@@ -30,18 +45,134 @@ const showMessage = (message, sender) => {
...
+
+// second question
+const petTypes = (name) => {
+  questionStep++
+  botAnswer(`Nice to meet you ${name} :) What kind of pet needs our care?`)
+
+  inputWrapper.innerHTML = `
+  <button id= 'dogButton'>dog 🐶</button>
+  <button id= 'catButton'>cat 🐱</button>
+  `
```
Comment (Svempolin) [inline code/script.js:94]: NIce, that you used dog and cat img :)

File: code/script.js [hunk 3]
```diff
@@ -30,18 +45,134 @@ const showMessage = (message, sender) => {
...
+  <video class="dog" loop autoplay playsinline>
+  <source src="assets/pets.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+  </video>
+  `)
+  } else {
+    botAnswer(`
+  <video class="cat" loop autoplay playsinline>
+  <source src="assets/cat-pretty.mp4" type="video/mp4">
+  Your browser does not support the video tag.
```
Comment (Svempolin) [inline code/script.js:115]: This was cool we could not fix the img in the chat!

File: code/script.js [hunk 4]
```diff
@@ -30,18 +45,134 @@ const showMessage = (message, sender) => {
...
+// forth question/confirmation from user
+const priceInformation = (choice) => {
+  questionStep++
+  if (choice === 'bathing') {
+    botAnswer(`So ${choice} is a great choice! We are ready to take care of your pet. That will cost 20$`);
+  } else if (choice === 'brushing') {
+    botAnswer(`Great, so you chose ${choice} service! We definitely can help with that! That will cost 15$`);
+  } else if (choice === 'nail trimming') {
+    botAnswer(`Wow, luxirous choice! ${choice} service will cost 13$`);
+  } else {
```
Comment (Svempolin) [inline code/script.js:143]: I really like this price Information!

File: code/script.js [hunk 5]
```diff
@@ -30,18 +45,134 @@ const showMessage = (message, sender) => {
...
+    botAnswer(`Great choice! We are the best in this! ⭐️ ${choice} service will cost 25$`)
+  }
+  inputWrapper.innerHTML = `
+  <button id='okButton'>OK</button>`
+
+  document.getElementById('okButton').addEventListener('click', () => usersPet('OK'))
```
Comment (themisk84) [inline code/script.js:149]: Nice structure!

File: code/script.js [hunk 6]
```diff
@@ -30,18 +45,134 @@ const showMessage = (message, sender) => {
...
+  document.getElementById('cardButton').addEventListener('click', () => usersPet('Card 💳'))
+}
+
+// goodbye from bot
+const petBye = () => {
+  botAnswer(`Thank you for your booking! We are looking forward to meet you and your pet.`)
+  botAnswer(`Have a nice day! 👋🏼`)
+  inputWrapper.innerHTML = ``
+}
+
```
Comment (Svempolin) [inline code/script.js:170]: Really nice structure

**Review by Svempolin — COMMENTED** [review]
We are really impressed of your work and style. Good work!!!!

File: code/script.js [hunk 7]
```diff
@@ -30,18 +45,134 @@ const showMessage = (message, sender) => {
...
+  </video>
+  `)
+  } else {
+    botAnswer(`
+  <video class="cat" loop autoplay playsinline>
+  <source src="assets/cat-pretty.mp4" type="video/mp4">
+  Your browser does not support the video tag.
+  </video>
+  `)
+  }
```
Comment (themisk84) [inline code/script.js:118]: Loved your videos!

**Review by themisk84 — APPROVED** [review]
Very nice job!Keep up the good work Julia and Mamite!

**Pull request closed without merging**